phd-tests/framework/src/disk/mod.rs

```diff
@@ -25,20 +26,30 @@ use crate::{
 use self::{crucible::CrucibleDisk, file::FileBackedDisk};
 
 pub mod crucible;
+pub mod fat;
 mod file;
+pub mod in_memory;
 
 /// Errors that can arise while working with disks.
 #[derive(Debug, Error)]
 pub enum DiskError {
     #[error("Disk factory has no Crucible downstairs artifact")]
     NoCrucibleDownstairs,
 
+    #[error(
+        "can't create a disk of the requested type with the requested source"
+    )]
+    SourceNotSupported,
```

Comment:
nitpicky, not super important: it might be nice if this error included the requested type and the requested source?

Reply:
Done in 85c0b95.